build/scenarios-efcore.sh

```diff
@@ -21,7 +21,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT=$DIR/..
 SESSION=`date '+%Y%m%d%H%M%S'`
 
-efcoreJobs="-r EntityFrameworkCore@release/3.1 --description Benchmarks --arg --perflab --no-global-json --server-timeout 00:45:00 -t EFCoreBenchmarks --aspnetcoreversion 3.1 --runtimeversion 3.1"
+efcoreJobs="-r EntityFrameworkCore@master --description Benchmarks --arg --perflab --no-global-json --server-timeout 00:45:00 -t EFCoreBenchmarks --aspnetcoreversion 5.0 --runtimeversion 5.0"
```

Comment:
You don't need the runtime and aspnet version arguments anymore then, they are that by default.

Reply:
That's great. Removed.